Microsoft.Toolkit.Uwp.UI.Controls/HamburgerMenu/HamburgerMenu.xaml

```diff
@@ -81,15 +87,70 @@
                                               AutomationProperties.Name="Content"
                                               Content="{TemplateBinding Content}" />
                         </SplitView>
-                        <Button Name="HamburgerButton"
-                                Width="{TemplateBinding HamburgerWidth}"
-                                Height="{TemplateBinding HamburgerHeight}"
-                                Padding="0"
-                                VerticalAlignment="Top"
-                                AutomationProperties.Name="Main button"
-                                Background="Transparent"
-                                BorderThickness="0"
-                                TabIndex="0">
+                        <Button 
+                            Name="HamburgerButton"
+                            Width="{TemplateBinding HamburgerWidth}"
+                            Height="{TemplateBinding HamburgerHeight}"
+                            Padding="0"
+                            VerticalAlignment="Top"
+                            AutomationProperties.Name="Main button"
+                            Background="{TemplateBinding AccentColor}"
```

Comment:
Hi @timheuer! I followed the practices used for the PaneBackground Property. There was no default set. Not in XAML nor in the Dependency property definition. If the property is unset, I expect that it will default to null which should resolve to Transparent brush being applied. This is what I observed during my manual testing.